package/MDAnalysis/lib/formats/include/readdcd.h

```diff
@@ -136,7 +136,7 @@ static int write_dcdheader(fio_fd fd, const char *remarks, int natoms,
  * Output: 0 on success, negative error code on failure.
  * Side effects: coordinates are written to the dcd file.
  */
-static int write_dcdstep(fio_fd fd, int curstep, int curframe, 
```

Comment:
@orbeckst Historical context? We have permission to use this or even a more recent version of the dcdplugin? To be fair, I didn't know there was an upstream change--I just thought it was logical and then looked it up just now. I'm also slightly confused that I have to hack the C / Cython to get unit cell writing to work--why isn't the C just handling that for the format it is supposed to support. Since MDA has needed this kind of thing previously, I suppose it is not too worrying.

It might be worth checking with John Stone about the changes to this plugin in the last 12 years & wrt to the unit cell handling. I'm assuming he was asked for permission at some stage a long time ago or the license is just open enough?!